data-migration/migrate-sharding-mysql-tidb-above-tb.md

```diff
@@ -0,0 +1,436 @@
+
+### Create the target schema
+
+After you make changes in the aforementioned [Check conflicts for sharded tables](data-migration\migrate-sharding-mysql-tidb-above-tb.md#check-conflicts-for-sharding-tables), you can now manually create the `my_db` schema and `table5` in downstream TiDB. After that, you need to configure `tidb-lightning.toml`.
+
+```toml
+[mydumper]
+no-schema = true # If you have created the downstream schema and tables, setting `true` tells TiDB Lightning not to create the downstream schema.
+```
+
+### Start the migration task
+
+Follow these steps to start `tidb-lightning`:
+
+1. Upload the exported full data to the server where TiDB Lightning is deployed.
+2. Edit the toml file. `tidb-lightning.toml` is used in the following example:
+
+    ```toml
+    [lightning]
+    # Logs
+    level = "info"
+    file = "tidb-lightning.log"
+
+    [tikv-importer]
+    # Choose a local backend.
+    # "local": The default mode. It is used for large data volumes greater than 1 TiB. During migration, downstream TiDB cannot provide services.
+    # "tidb": Used for data volumes less than 1 TiB. During migration, downstream TiDB can provide services normally.
+    # For more information, see [TiDB Lightning Backends](https://docs.pingcap.com/tidb/stable/tidb-lightning-backends)
+    backend = "local"
+    # Set the temporary directory for the sorted key value pairs. It must be empty.
+    # The free space must be greater than the largest single table of the data source.
+    # It is recommended that you use a directory different from `data-source-dir` to get better migration performance by consuming IO resources exclusively.
+    sorted-kv-dir = "${sorted-kv-dir}"
+
+    # Set the renaming rules ('routes') from source to target tables, in order to support merging different table shards into a single target table. Here you migrate `table1` and `table2` in `my_db1`, and `table3` and `table4` in `my_db2`, to the target `table5` in downstream `my_db`.
+    [[routes]]
+    schema-pattern = "my_db1"
+    table-pattern = "table[1-2]"
+    target-schema = "my_db"
+    target-table = "table5"
+
+    [[routes]]
+    schema-pattern = "my_db2"
+    table-pattern = "table[3-4]"
+    target-schema = "my_db"
+    target-table = "table5"
+
+    [mydumper]
+    # The source data directory. Set this to the path of the Dumpling exported data.
+    # If there are several Dumpling-exported data directories, you need to place all these directories in the same parent directory, and use the parent directory here.
+    data-source-dir = "${data-path}"
+    # Because table1~table4 from source are merged into another table5 in the target, you should tell TiDB Lightning no need to create schemas, so that table1 ~ table4 won't be created automatically according to the exported schema information
+    # This parameter can prevent creating tables1~4 downstream based on files exported by Dumpling.
```

Comment:
```suggestion
    # Because table1~table4 from source are merged into another table5 in the target, you should tell TiDB Lightning no need to create schemas, so that table1 ~ table4 won't be created automatically according to the exported schema information.
```